platform/pddf/i2c/utils/pddf_util.py

```diff
@@ -486,60 +490,60 @@ def do_switch_pddf():
     try:
         import pddf_switch_svc
     except ImportError:
-        print "Unable to find pddf_switch_svc.py. PDDF might not be supported on this platform"
+        print("Unable to find pddf_switch_svc.py. PDDF might not be supported on this platform")
         sys.exit()
-    print "Check the pddf support..."
+    print("Check the pddf support...")
     status = pddf_switch_svc.check_pddf_support()
     if not status:
-        print "PDDF is not supported on this platform"
+        print("PDDF is not supported on this platform")
         return status
 
 
-    print "Checking system...."
+    print("Checking system....")
     if os.path.exists('/usr/share/sonic/platform/pddf_support'):
-        print PROJECT_NAME.upper() +" system is already in pddf mode...."
+        print(PROJECT_NAME.upper() +" system is already in pddf mode....")
     else:
-        print "Check if the native sonic-platform whl package is installed in the pmon docker"
-        status, output = log_os_system("docker exec -it pmon pip show sonic-platform", 1)
+        print("Check if the native sonic-platform whl package is installed in the pmon docker")
+        status, output = log_os_system("docker exec -it pmon pip3 show sonic-platform", 1)
         if not status:
             # Need to remove this whl module
-            status, output = log_os_system("docker exec -it pmon pip uninstall sonic-platform -y", 1)
+            status, output = log_os_system("docker exec -it pmon pip3 uninstall sonic-platform -y", 1)
             if not status:
-                print "Successfully uninstalled the native sonic-platform whl pkg from pmon container"
+                print("Successfully uninstalled the native sonic-platform whl pkg from pmon container")
             else:
-                print "Error: Unable to uninstall the sonic-platform whl pkg from pmon container.\
+                print("Error: Unable to uninstall the sonic-platform whl pkg from pmon container.\)
```

Comment:
LGTM caught this syntax error. Closing parenthesis should be placed at the end of the next line and the backslash can be removed.